torchinductor/config.py

```diff
@@ -71,7 +71,7 @@ class triton:
 
     # limit tiling dimensions
     # Disable tiling until we figure out how tiling and fusion work together
-    max_tiles = 1
```

Comment:
Should we keep `max_tiles = 1` for fusion?

Reply:
If this makes things faster (or helps your testing) feel free to change it.